2022-03-23 04:05:40,649 INFO    MainThread:1393 [wandb_setup.py:_flush():69] setting env: {}
2022-03-23 04:05:40,649 INFO    MainThread:1393 [wandb_setup.py:_flush():69] setting login settings: {}
2022-03-23 04:05:40,649 INFO    MainThread:1393 [wandb_init.py:_log_setup():336] Logging user logs to /workspace/graph-neural-pde/src/wandb/run-20220323_040540-vanilla alpha_ = 0.9 alpha = 100 dopri5 test/logs/debug.log
2022-03-23 04:05:40,650 INFO    MainThread:1393 [wandb_init.py:_log_setup():337] Logging internal logs to /workspace/graph-neural-pde/src/wandb/run-20220323_040540-vanilla alpha_ = 0.9 alpha = 100 dopri5 test/logs/debug-internal.log
2022-03-23 04:05:40,650 INFO    MainThread:1393 [wandb_init.py:init():369] calling init triggers
2022-03-23 04:05:40,650 INFO    MainThread:1393 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-03-23 04:05:40,650 INFO    MainThread:1393 [wandb_init.py:init():418] starting backend
2022-03-23 04:05:40,650 INFO    MainThread:1393 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-23 04:05:40,664 INFO    MainThread:1393 [backend.py:ensure_launched():123] starting backend process...
2022-03-23 04:05:40,672 INFO    MainThread:1393 [backend.py:ensure_launched():128] started backend process with pid: 1419
2022-03-23 04:05:40,674 INFO    MainThread:1393 [wandb_init.py:init():423] backend started and connected
2022-03-23 04:05:40,677 INFO    MainThread:1393 [wandb_init.py:init():465] updated telemetry
2022-03-23 04:05:40,678 INFO    MainThread:1393 [wandb_init.py:init():484] communicating current version
2022-03-23 04:05:41,897 INFO    MainThread:1393 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-23 04:05:41,898 INFO    MainThread:1393 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-03-23 04:05:42,413 INFO    MainThread:1393 [wandb_init.py:init():522] starting run threads in backend
2022-03-23 04:05:44,783 INFO    MainThread:1393 [wandb_run.py:_console_start():1524] atexit reg
2022-03-23 04:05:44,784 INFO    MainThread:1393 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2022-03-23 04:05:44,786 INFO    MainThread:1393 [wandb_run.py:_redirect():1376] Redirecting console.
2022-03-23 04:05:44,788 INFO    MainThread:1393 [wandb_run.py:_redirect():1438] Redirects installed.
2022-03-23 04:05:44,789 INFO    MainThread:1393 [wandb_init.py:init():546] run started, returning control to user process
2022-03-23 04:05:45,518 INFO    MainThread:1393 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2022-03-23 04:05:45,520 INFO    MainThread:1393 [wandb_run.py:_restore():1466] restore
2022-03-23 04:05:48,139 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 895
  total_bytes: 895
}

2022-03-23 04:05:48,448 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 895
  total_bytes: 6935
}

2022-03-23 04:05:49,103 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 895
  total_bytes: 6935
}

2022-03-23 04:05:49,207 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:49,312 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:49,417 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:49,523 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:49,626 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:49,729 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:49,834 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:49,936 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:50,038 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:50,145 INFO    MainThread:1393 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6935
  total_bytes: 6935
}

2022-03-23 04:05:51,451 INFO    MainThread:1393 [wandb_run.py:_show_files():1838] logging synced files
